src/Sylius/Bundle/AttributeBundle/Resources/config/attribute_types.xml

```diff
@@ -100,6 +103,15 @@
         <service id="sylius.form.type.attribute_type.configuration.date" class="%sylius.form.type.attribute_type.configuration.date.class%">
             <tag name="form.type" alias="sylius_attribute_type_configuration_date" />
         </service>
+        <service id="sylius.attribute_type.select" class="%sylius.attribute_type.select.class%">
+            <tag name="sylius.attribute.type" attribute-type="select" label="Select" />
```

Comment:
For now all labels are unfortunately hardcodes, but it surely should be changed in separate PR (if it **can** be changed :smile:)